src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/EmptyArray.cs

```diff
@@ -5,6 +5,6 @@ namespace System.Reflection.Internal
 {
     internal static class EmptyArray<T>
     {
-        internal static readonly T[] Instance = new T[0];
+        internal static readonly T[] Instance = Array.Empty<T>();
```

Comment:
Does this work?  I thought this assembly built against an earlier contract that lacks Empty.